src/main/scala/cromwell/engine/backend/sge/SgeBackend.scala

```diff
@@ -77,11 +77,10 @@ class SgeBackend extends Backend with SharedFileSystem with LazyLogging {
     globalDataAccess.updateExecutionBackendInfo(call.workflowDescriptor.id, CallKey(call.call, call.key.index), backendInfo)
   }
 
-  // TODO: Not much thought was given to this function
-  override def handleCallRestarts(restartableWorkflows: Seq[RestartableWorkflow])
-                                 (implicit ec: ExecutionContext): Future[Any] = {
-    Future.successful(Unit)
-  }
+  /** TODO restart isn't currently implemented for SGE, there is probably work that needs to be done here much like
```

Comment:
Is this really scaladoc or is it just a multi-line comment (i.e. `/*`).

^^^ That might be the most pedantic code review comment i've ever made